lib/main.js

```diff
@@ -272,7 +272,7 @@ Lambda.prototype._rsync = function (program, src, dest, excludeNodeModules, call
     }
 
     // include package.json unless prebuiltDirectory is set
-    var includeArgs = program.prebuiltDirectory ? '' : '--include package.json ';
+    var includeArgs = program.prebuiltDirectory ? '' : '--include /package.json ';
```

Comment:
The `include` and `exclude` options of rsync are complicated, so I can not explain the exact behavior, but the result was as follows.

# src directory
```
% pwd
/tmp/test
% find /tmp/test -type f
/tmp/test/package.json
/tmp/test/foo/package.json
```
# exclude='foo/*'
## include='package.json'
```
% rm -rf /tmp/bar; rsync -rL --include='package.json' --exclude='foo/*' /tmp/test /tmp/bar
% find /tmp/bar -type f
/tmp/bar/test/package.json
/tmp/bar/test/foo/package.json
```

## include='/package.json'
```
% rm -rf /tmp/bar; rsync -rL --include='/package.json' --exclude='foo/*' /tmp/test /tmp/bar
% find /tmp/bar -type f
/tmp/bar/test/package.json
```

# exclude='foo'
## include='package.json'
```
% rm -rf /tmp/bar; rsync -rL --include='package.json' --exclude='foo' /tmp/test /tmp/bar  
% find /tmp/bar -type f
/tmp/bar/test/package.json
```

## include='/package.json'
```
% rm -rf /tmp/bar; rsync -rL --include='/package.json' --exclude='foo' /tmp/test /tmp/bar
% find /tmp/bar -type f
/tmp/bar/test/package.json
```